module-4/backend/week_two.md

```diff
@@ -5,18 +5,55 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's one difference between ES5 and ES6?
+
+fat arrow functions. using let and const.
+
 2. What's the difference between asynchronous and synchronous JavaScript? 
+
+synchronous means everything happens in a fixed order where asychronous means that events can happen any order.
```

Comment:
I would go into more detail about both - what do you mean by 'fixed order' and ' can happen any order' - or be ready for an interviewer to push you to go deeper if you start by answering at this higher level.